packages/wdio-cli/src/launcher.js

```diff
@@ -97,20 +97,10 @@ class Launcher {
      * run without triggering onPrepare/onComplete hooks
      */
     runMode (config, caps) {
-        /**
-         * if it is an object run multiremote test
-         */
-        if (this.isMultiremote) {
-            return new Promise((resolve) => {
-                this.resolve = resolve
-                this.startInstance(this.configParser.getSpecs(), caps, 0)
-            })
-        }
-
         /**
          * fail if no caps were found
          */
-        if (!caps || !caps.length) {
+        if (!caps || (!this.isMultiremote && !caps.length)) {
```

Comment:
can be simplified to:
`if (!caps || !caps.length)`

Reply:
`caps` is an object in Multiremote, while in regular mode it is an array.

Before current PR this block was unreachable in Multiremote, so have to add this condition